sources/packages/web/src/components/institutions/confirmation-of-enrollment/modals/ConfirmCOEDenyModal.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <ModalDialogBase
+    title="Decline Confirmation of Enrolment"
+    dialogType="question"
+    :showDialog="showDialog"
+    @dialogClosed="dialogClosed"
+  >
+    <template v-slot:content>
+      <v-container>
+        <formio
+          formName="declineconfirmationofenrollment"
+          @loaded="formLoaded"
+          @submitted="submitApplication"
+        ></formio>
+      </v-container>
+    </template>
+    <template v-slot:footer>
+      <v-btn color="primary" outlined @click="dialogClosed">
+        Cancel
+      </v-btn>
+      <v-btn
+        color="danger"
+        depressed
+        @click="denyProgramInfo"
+        style="color:white"
+      >
+        <v-icon left size="25">
+          mdi-cancel
+        </v-icon>
+        Decline Request
+      </v-btn>
+    </template>
+  </ModalDialogBase>
+</template>
+
+<script lang="ts">
+import { ref } from "vue";
+import ModalDialogBase from "@/components/generic/ModalDialogBase.vue";
+import { useModalDialog, useFormioUtils } from "@/composables";
+import formio from "@/components/generic/formio.vue";
+import { ConfirmationOfEnrollmentService } from "@/services/ConfirmationOfEnrollmentService";
+export const COE_DENIAL_REASON_RADIO = "coeDenyReasonId";
```

Comment:
Since this const is not being used anywhere else besides this class, it does not need to be exported.